plugins/mcreator-localization/help/fr_FR/particle/texture.md

```diff
@@ -1,4 +1,4 @@
-Si la texture est une texture tiled (plusieurs textures dans une bande verticale),
+Si la texture est une texture carrelée (plusieurs textures dans une bande verticale),

```

Comment:
```suggestion
Si la texture est une texture tiled (plusieurs textures dans une bande verticale),
```

Reply:
Feel free to open PR to fix this it should be insta-merged